contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -3632,27 +3632,11 @@ execute_bulk_load_insert(int ncol, int nrow,
 		 * In an error condition, the caller calls the function again to do
 		 * the cleanup.
 		 */
-		MemoryContext oldcontext;
```

Comment:
should we reset offset here?

Reply:
No, we do implicit batching and processing in middle of TDS packet, thats why we needed offset to be store and carry own reading from next